GameOfLifeApp/GameLogic.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLifeApp
+{
+    public class GameLogic
+    {
+        bool[,] generatedField;
+
+        /// <summary>
+        /// Holds user data of array size.    
```

Comment:
If you add summaries for properties, would be better to add them for all the properties, not just some random ones.

Reply:
Added.